### PR TITLE
account: Choose `User` is the default selection when the user role is Domain Admin

### LINKED ISSUE #746: [FEATURE] domain admin should have "User" as default account type when adding an account
**Is your feature request related to a problem? Please describe.**
As a Domain Admin my ussual account type to add is "User" so I want this to be the default choice.

image

### PR DESCRIPTION
**Pull request by utchoang**
Fixes #746 
@rhtyd cc @svenvogel 
image

### CODE REVIEW

**Comment by utchoang** [comment]
@blueorangutan package

File: src/views/iam/AddAccount.vue [hunk 1]
```diff
@@ -311,6 +315,12 @@ export default {
       api('listRoles').then(response => {
         this.roles = response.listrolesresponse.role || []
         this.selectedRole = this.roles[0].id
+        if (this.isDomainAdmin) {
+          const domainAdminRole = this.roles.filter(role => role.type === 'User')
```
Comment (davidjumani) [inline src/views/iam/AddAccount.vue:319]: @utchoang Could this be better renamed to userRole ?

**Comment by davidjumani** [comment]
@blueorangutan package

**Review by davidjumani — APPROVED** [review]
LGTM! Thanks @utchoang

**Pull request merged**